fuel-relayer/flake.nix

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
can we have some tips on how to use this added to fuel-relayer readme.md?

Reply:
Will do, I'm just working on making it a little easier to use